metricflow/validation/dataflow_join_validator.py

```diff
@@ -40,13 +40,13 @@ def is_valid_instance_set_join(
         left_instance_set: InstanceSet,
         right_instance_set: InstanceSet,
         on_entity_reference: EntityReference,
-        right_node_is_subquery: bool = False,
+        right_node_is_aggregated_to_entity: bool = False,
```

Comment:
Oh neat, so we can (probably) swap this to the elements and do a straight comparison later.